src/displaysize.rs

```diff
@@ -1,31 +1,101 @@
 //! Display size
 
-// TODO: Add to prelude
-/// Display size enumeration
-#[derive(Clone, Copy)]
-pub enum DisplaySize {
-    /// 128 by 64 pixels
-    Display128x64,
-    /// 128 by 32 pixels
-    Display128x32,
-    /// 96 by 16 pixels
-    Display96x16,
-    /// 70 by 42 pixels
-    Display72x40,
-    /// 64 by 48 pixels
-    Display64x48,
+// No need to implement Copy
+#![allow(missing_copy_implementations)]
```

Comment:
I agree it's a bit superfluous, but it can't hurt to add `#[derive(Debug, Copy, Clone)]` to all the size structs IMO. Someone out there might find a use for it :)